frontend/app/controllers/create-badges.js

```diff
@@ -40,19 +40,19 @@ export default Controller.extend({
   csvEnable      : false,
   manualEnable   : false,
   defImage       : false,
-  custImage      : false,
+  custImage      : true,
   colorImage     : false,
   overlay        : false,
   showProgress   : false,
   imageData      : null,
   progress       : 0,
   progressState  : '',
-  firstName      : '',
-  lastName       : '',
-  organization   : '',
-  socialHandle   : '',
-  designation    : '',
-  prevImageData  : '',
+  firstName      : 'Dominic',
```

Comment:
@gabru-md  If we can have just  `First Name`, `Last Name` and so on in the default preview. This is what we have discussed in Meeting.

Reply:
I guess this looks a lot better than writing `First Name`, `Last Name` etc....